Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/InBetween.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Utilities.Solvers
+{
+    /// <summary>
+    /// InBetween solver positions an object in-between two tracked transforms.
+    /// </summary>
+    public class InBetween : Solver
+    {
+        [SerializeField]
+        [Tooltip("Distance along the center line the object will be located. 0.5 is halfway, 1.0 is at the second transform, 0.0 is at the first transform.")]
+        [Range(0f, 1f)]
+        private float partwayOffset = 0.5f;
+
+        /// <summary>
+        /// Distance along the center line the object will be located. 0.5 is halfway, 1.0 is at the second transform, 0.0 is at the first transform.
+        /// </summary>
+        public float PartwayOffset
+        {
+            get { return partwayOffset; }
+            set { partwayOffset = Mathf.Clamp(value, 0.0f, 1.0f); }
+        }
+
+        [SerializeField]
+        [Tooltip("Tracked object to calculate position and orientation for the second object. If you want to manually override and use a scene object, use the TransformTarget field.")]
+        [HideInInspector]
```

Comment:
Why have this attibute when we have a custom inspector?

Reply:
You didn't like the way we were hiding this field in the original custom inspector when this was added to master in #2245, so this tag was added. I'm happy to revert.